### PR TITLE
Issue #93 -Fix count of registrated people.

### LINKED ISSUE #93: Registration
- [x] kontrola konca registracie
- [x] registruj attendee s existujucim emailom s novym menom
- [x] fix count() on count fo registrated people
#17

### PR DESCRIPTION
**Pull request by nitrajka**
Resolves #93

### CODE REVIEW

File: src/src/AppBundle/Controller/DefaultController.php [hunk 1]
```diff
@@ -87,44 +85,59 @@ public function indexAction($slug, $_locale, Request $request)
...
             $firstName = $form->getData()['first_name'];
             $lastName = $form->getData()['last_name'];
+            $event = $em->getRepository(Event::class)->find($form->getData()['event_choice']);
+
+            $attendee = $em->getRepository(Attendee::class)->findOneBy(array('email' => $email));
+            if ($attendee) {
+                $attendeeRegistratedForEvent = $em->getRepository(Registration::class)->findOneBy(
+                                                        ['attendee'=> $attendee, 'events' => $event]);
+                if ($attendeeRegistratedForEvent) { // attendee already registered for event
+                    // todo: Viki: problem je v tom, ze view si uz vytvorila a teda sa tato chyba vo view neukaze.
```
Comment (ZoltanOnody) [inline src/src/AppBundle/Controller/DefaultController.php]: todo uz mozes zmazat asi.

File: src/src/AppBundle/Controller/DefaultController.php [hunk 2]
```diff
@@ -87,44 +85,59 @@ public function indexAction($slug, $_locale, Request $request)
...
+                    $form->get('email')->addError(new FormError('Attendee already registrated for that event.'));
```
Comment (ZoltanOnody) [inline src/src/AppBundle/Controller/DefaultController.php]: prosim ta, vsetky tieto texty tahaj z `lang` suborov, 

`$context['lang']['already_registered']`

a podobne, nech sa daju potom menit, oki?

**Review by ZoltanOnody — CHANGES_REQUESTED** [review]

File: src/src/AppBundle/Controller/DefaultController.php (hunk 1 above)
Comment (nitrajka) [reply src/src/AppBundle/Controller/DefaultController.php]: Oki

File: src/src/AppBundle/Controller/DefaultController.php (hunk 2 above)
Comment (nitrajka) [reply src/src/AppBundle/Controller/DefaultController.php]: Oki, idem to este opravit.

**Review by ZoltanOnody — DISMISSED** [review]
otherwise LGTM

**Review by ZoltanOnody — APPROVED** [review]

**Pull request merged**